openmdao/visualization/connection_viewer/viewconns.py

```diff
@@ -206,41 +206,53 @@ def view_connections(root, outfile='connections.html', show_browser=True,
         'show_values': show_values,
     }
 
-    viewer = 'connect_table.html'
+    if '.html' in outfile:
```

Comment:
These if statements should probably look like:
```
if outfile.endswith('.html'):
    ...
elif outfile.endswith('.csv'):
    ...
else
    raise RuntimeError("Invalid file extension for output file, should be '.html' or '.csv'")
```
(and per Ken's comment, there should be test coverage for the two new branches)